### PR TITLE
...since it's also valid for IM

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

**Review by PennyWalker — COMMENTED** [review]
I think it's a good change. This is a very definitive statement, if I was doing it, I would have changed to "Do you know that people can misunderstand sarcasm in text."

File: rules/do-you-know-that-people-misunderstand-sarcasm-in-email/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Do you know that people misunderstand sarcasm in email?
+title: Do you know that people misunderstand sarcasm in writing?
```
Comment (bradystroud) [inline rules/do-you-know-that-people-misunderstand-sarcasm-in-email/rule.md:4]: I agree with Penny, this is a good change but "Do you know that people can misunderstand sarcasm in text?" is even better.
```suggestion
title: Do you know that people misunderstand sarcasm in text?
```

**Review by UlyssesMaclaren — COMMENTED** [review]
good change... btw apparently this is the sarcasm emoji: 🤌

**Comment by tiagov8** [comment]
Hey @bradystroud 

Please review squash and merge with the new title:

> Do you know that people misunderstand sarcasm in text?

**Review by bradystroud — APPROVED** [review]

**Pull request merged**